TelemetryEvents.swift

```diff
@@ -0,0 +1,45 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+import Shared
+
+private let PrefKeySearches = "Telemetry.Searches"
+
+public class SearchTelemetryFactory {
+    // For data consistency, the strings used here are identical to the ones reported in Android.
+    public enum Source: String {
+        case URLBar = "actionbar"
```

Comment:
Interesting! I wasn't aware we overrode the `toString` method to return these values. (fwiw, it's wasteful when you already have the name in `Enum.name()` but I suppose those values are much less obnoxious!)

Carry on – I'll fix that in Java.